### ISSUE TITLE
To add ability to list files to check

### ISSUE THREAD

**Issue by sergio-fry** [turboladen/tailor#83]
Instead of one file or all .rb files inside path. The idea is to check only those files, that have been updated.

**Comment by turboladen** [comment]
Good idea.  I also like the idea of doing something along the autotest or https://github.com/mynyml/watchr lines.

**Comment by sergio-fry** [comment]
Agreed! I think it is an idea for a new new gem 'auto-tailor' :)

**Comment by turboladen** [comment]
Added an issue for the auto-test idea: https://github.com/turboladen/tailor/issues/88

**Comment by turboladen** [comment]
This should be fixed as of e526610f75fb3bea15ad6bce03243f067b132e4e.  Expect in 1.0.0.

**Comment by turboladen** [comment]
I misunderstood the problem when I closed this.  I understood that you'd asked to be able to specify a list of files that tailor should check (like in a configuration file or something)--that has been implemented.  I just realized that that's not what this issue is requesting.  Reopening accordingly.
